packages/css/src/ordered-list/ordered-list.scss

```diff
@@ -5,39 +5,54 @@
 
 @import "../../utils/breakpoint";
 
+/**
+ * - Applies border box sizing.
+ * - Resets whitespace and list styles.
+ */
 @mixin reset {
   box-sizing: border-box;
+  list-style-type: none;
   margin-block: 0;
+  padding-inline-start: 0;
   -webkit-text-size-adjust: 100%;
 }
 
+/** A list is a grid column with a gap between items. */
 .amsterdam-ordered-list {
+  @include reset;
+
+  display: grid;
+  gap: var(--amsterdam-ordered-list-gap);
+}
+
+/**
+ * A list with markers:
+ * - sets typography for its text content
+ * - applies correct spacing at the start of items.
+ */
+.amsterdam-ordered-list:not(.amsterdam-ordered-list--no-markers) {
   color: var(--amsterdam-ordered-list-color);
-  display: flex;
-  flex-direction: column;
   font-family: var(--amsterdam-ordered-list-font-family);
   font-size: var(--amsterdam-ordered-list-narrow-font-size);
   font-weight: var(--amsterdam-ordered-list-font-weight);
-  gap: var(--amsterdam-ordered-list-gap);
   line-height: var(--amsterdam-ordered-list-line-height);
+  list-style-type: var(--amsterdam-ordered-list-list-style-type);
 
   @media screen and (width > $amsterdam-breakpoint) {
     font-size: var(--amsterdam-ordered-list-wide-font-size);
   }
 
-  @include reset;
-}
-
-.amsterdam-ordered-list__item {
-  padding-inline: var(--amsterdam-ordered-list-item-padding-inline);
+  li {
+    margin-inline-start: var(--amsterdam-ordered-list-item-margin-inline-start);
+    padding-inline-start: var(--amsterdam-ordered-list-item-padding-inline-start);
+  }
 }
 
+/** A nested list has a different marker and less spacing for correct alignment. */
 .amsterdam-ordered-list .amsterdam-ordered-list {
   list-style-type: var(--amsterdam-ordered-list-ordered-list-list-style-type);
-  padding-block: var(--amsterdam-ordered-list-ordered-list-padding-block);
```

Comment:
Shouldn't there be `0.5rem` spacing between an item's text and its nested list as well?

Reply:
No, I don’t think so. That would position a nested list exactly between its parent item and the next parent item.

Keeping it a bit closer to its parent further indicates where it belongs.

I’ll add this to a task to feedback the designers.

Reply:
Hm, I'm not a huge fan of how that looks, but let's discuss it with the designers